test/server/enrichment-map/enrichment/enrichment-test.js

```diff
@@ -1,90 +1,102 @@
-           't depth': '1',
-           'Q&T list': 'AFF4' },
+          {
+            signf: '!',
+            pvalue: '8.70e-03',
+            T: '100',
+            Q: '1',
+            'Q&T': '1',
+            'Q&T/Q': '1.000',
+            'Q&T/T': '0.010',
+            't type': 'BP',
+            't group': '3',
+            't name': '   transcription elongation from RNA polymerase II promoter',
+            't depth': '1',
+            'Q&T list': 'AFF4'
+          },
         'REAC:75955':
-         { signf: '!',
-           pvalue: '7.75e-03',
-           T: '61',
-           Q: '1',
-           'Q&T': '1',
-           'Q&T/Q': '1.000',
-           'Q&T/T': '0.016',
-           't type': 'rea',
-           't group': '5',
-           't name': '   RNA Polymerase II Transcription Elongation',
-           't depth': '1',
-           'Q&T list': 'AFF4' },
+          {
+            signf: '!',
+            pvalue: '7.75e-03',
+            T: '61',
+            Q: '1',
+            'Q&T': '1',
+            'Q&T/Q': '1.000',
+            'Q&T/T': '0.016',
+            't type': 'rea',
+            't group': '5',
+            't name': '   RNA Polymerase II Transcription Elongation',
+            't depth': '1',
+            'Q&T list': 'AFF4'
+          },
         'REAC:112382':
-         { signf: '!',
-           pvalue: '7.75e-03',
-           T: '61',
-           Q: '1',
-           'Q&T': '1',
-           'Q&T/Q': '1.000',
-           'Q&T/T': '0.016',
-           't type': 'rea',
-           't group': '5',
-           't name': '    Formation of RNA Pol II elongation complex ',
-           't depth': '2',
-           'Q&T list': 'AFF4' },
+          {
+            signf: '!',
+            pvalue: '7.75e-03',
+            T: '61',
+            Q: '1',
+            'Q&T': '1',
+            'Q&T/Q': '1.000',
+            'Q&T/T': '0.016',
+            't type': 'rea',
+            't group': '5',
+            't name': '    Formation of RNA Pol II elongation complex ',
+            't depth': '2',
+            'Q&T list': 'AFF4'
+          },
         'REAC:674695':
-         { signf: '!',
-           pvalue: '7.75e-03',
-           T: '83',
-           Q: '1',
-           'Q&T': '1',
-           'Q&T/Q': '1.000',
-           'Q&T/T': '0.012',
-           't type': 'rea',
-           't group': '2',
-           't name': '   RNA Polymerase II Pre-transcription Events',
-           't depth': '1',
-           'Q&T list': 'AFF4' } };
-        expect(objectEquality(result, res)).to.equal(true);
+          {
+            signf: '!',
+            pvalue: '7.75e-03',
+            T: '83',
+            Q: '1',
+            'Q&T': '1',
+            'Q&T/Q': '1.000',
+            'Q&T/T': '0.012',
+            't type': 'rea',
+            't group': '2',
+            't name': '   RNA Polymerase II Pre-transcription Events',
+            't depth': '1',
+            'Q&T list': 'AFF4'
```

Comment:
Is this the expected format of the response?  These are not proper variable names...

Reply:
We only check for specific keys here. renamed the function to 'gProfilerResEquality'.